src/sass/_theme_rst.sass

```diff
@@ -85,18 +85,13 @@
     padding: ($base-line-height / 2) ($base-line-height / 2)
     display: block
     overflow: auto
-    & .hll
-      // Line emphasis spans full width
-      display: block
-      margin: 0 (-1 * $base-line-height / 2)
-      padding: 0 ($base-line-height / 2)
   pre.literal-block, div[class^='highlight'] pre, .linenodiv pre
     font-family: $code-font-family
     font-size: 12px
     line-height: 1.4
   
-  // Do not allow selection of code block prompts
-  div.highlight .gp 
+  // Do not allow selection of code block prompts and line numbers
+  .lineno, .gp 
```

Comment:
Noted some potential issues with the code blocks in Sphinx 4 output, though not sure any of this would be suspect.

Reply:
I fixed the issue with selecting line numbers. In sphinx4 the default for ` html_codeblock_linenos_style` is now inline which is why your other screenshot is different.

` html_codeblock_linenos_style` is also now depricated so I guess this is how sphinx is intending codeblocks to look from now on.

Reply:
I was still noticing the issues with selecting line numbers. I resolve the display inconsistencies with 6597b25